example/use_cases/employer_university_flow/employer_university_flow/util.go

```diff
@@ -0,0 +1,315 @@
+			Format:         claimFormat,
+			AlgOrProofType: claim.SignatureAlgorithmOrProofType,
+		})
+	}
+	return normalizedClaims
+}
+
+func BuildPresentationSubmission(presentationRequest []byte, signer cryptosuite.Signer, verifier cryptosuite.JSONWebKeyVerifier, vc credential.VerifiableCredential) ([]byte, error) {
+	return buildPresentationSubmission(presentationRequest, signer, verifier, vc)
+}
+
+// https://github.com/TBD54566975/ssi-sdk/blob/d279ca2779361091a70b8aa3c685a388067409a9/credential/exchange/submission.go#L126
+//
+func buildPresentationSubmission(presentationRequest []byte, signer cryptosuite.Signer, verifier cryptosuite.JSONWebKeyVerifier, vc credential.VerifiableCredential) ([]byte, error) {
+
+	presentationClaim := exchange.PresentationClaim{
+		Credential:                    &vc,
+		LDPFormat:                     exchange.LDPVC.Ptr(),
+		SignatureAlgorithmOrProofType: string(cryptosuite.JSONWebSignature2020),
+	}
+
+	parsed, err := verifier.VerifyAndParseJWT(string(presentationRequest))
+	if err != nil {
+		return nil, err
+	}
+
+	def, ok := parsed.Get(exchange.PresentationDefinitionKey)
+	if !ok {
+		return nil, fmt.Errorf("presentation definition key<%s> not found in token", exchange.PresentationDefinitionKey)
+	}
+
+	dat, err := json.Marshal(def)
+	if err != nil {
+		return nil, err
+	}
+	var pd exchange.PresentationDefinition
+	err = json.Unmarshal(dat, &pd)
+	if err != nil {
+		return nil, err
+	}
+
+	submissionBytes, err := exchange.BuildPresentationSubmission(signer, pd, []exchange.PresentationClaim{presentationClaim}, exchange.JWTVPTarget)
+	if err != nil {
+		return nil, err
+	}
+
+	return submissionBytes, nil
+}
+
+func MakePresentationData(id string, inputID string) (exchange.PresentationDefinition, error) {
+	return makePresentationData(id, inputID)
+}
+
+// Makes a dummy presentation definition. These are
+// eventually transported via Presentation Request.
+// For more information on presentation definitions go
+// https://identity.foundation/presentation-exchange/#term:presentation-definition
+func makePresentationData(id string, inputID string) (exchange.PresentationDefinition, error) {
+	// Input Descriptors: Describe the information the verifier requires of the holder
+	// https://identity.foundation/presentation-exchange/#input-descriptor
+	// Required fields: ID and Input Descriptors
+	def := exchange.PresentationDefinition{
+		ID: id,
+		InputDescriptors: []exchange.InputDescriptor{
+			{
+				ID: inputID,
+				Constraints: &exchange.Constraints{
+					Fields: []exchange.Field{
+						{
+							Path:    []string{"$.vc.issuer", "$.issuer"},
+							ID:      "issuer-input-descriptor",
+							Purpose: "need to check the issuer",
+						},
+					},
+				},
+			},
+		},
+	}
+	cw.WriteNote("Presentation Definition is formed. Asks for the issuer and the data from the issuer")
+	err := def.IsValid()
+	return def, err
+}
+
+func handleError(err error) {
+	if err != nil {
+		cw.WriteError(err.Error())
```

Comment:
this assumes a CW is initialized, which is risky. perhaps this should take a cw or be a reciever on a CW